src/screens/Signup/Signup.tsx

```diff
@@ -127,15 +128,18 @@ function Signup({ navigation }) {
 
       try {
         delete formData.confirmPassword;
-        await API.addUser(formData);
+        await API.addUser({ ...formData, events: [] });
       } catch (err) {
         Alert.alert('There was some error in saving user data.');
         return;
       }
 
       // emptying erros after successful verification
       setErrors({});
-      Alert.alert('Signup successful');
+
+      Alert.alert('Signup successful. Please login to continue');
```

Comment:
if i am not wrong when user signing up you were already logging in that user then why this prompt